index.html

```diff
@@ -43,9 +43,10 @@
             openmct.legacyRegistry.enable.bind(openmct.legacyRegistry)
         );
 
+        openmct.install(openmct.plugins.Elasticsearch('http://localhost:9200'));
```

Comment:
Please revert this change, Open MCT should use LocalStorage by default

Reply:
done